src/redux/fetch.js

```diff
@@ -0,0 +1,67 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const baseURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const uniqueId = 'FuHmnCA7EBqJ1jB7klF3';
+const URL = `${baseURL}/apps/${uniqueId}/books`;
+
+const initialState = {
+  books: [],
+  loading: false,
+};
+
+const GET_ALL_BOOKS = 'books/getAllBooks';
+const ADD_BOOK_API = 'book/addBookAPI';
+const REMOVE_BOOK_API = 'book/removeBookAPI';
+
+export const getAllBooks = createAsyncThunk(GET_ALL_BOOKS, async () => {
+  const respo = await axios.get(URL);
+  const { data } = respo;
+  const books = Object.entries(data).map(([key, value]) => {
+    const { title, category, author } = value[0];
+    return {
+      Id: key,
+      title,
```

Comment:
- [ ] Great job everything works as expected, but why did you make a whole new bookstore to interact with the API? Please add all the changes you made to `./src/redux/books/booksSlice.js` file and remove the newly created `./src/redux/fetch.js` file to maintain a professional file structure inside your app